templates/download/server/manual.html

```diff
@@ -166,9 +166,119 @@ <h2>Ubuntu {{ releases.lts.full_version }} LTS</h2>
         </div>
 
         <div class="p-tabs__content p-section"
-             id="how-to-install"
+             id="how-to-install-lts"
              role="tabpanel"
-             aria-labelledby="how-to-install-tab">
+             aria-labelledby="how-to-install-tab-lts">
+          <p>To install Ubuntu Server:</p>
+          <ol class="p-list--divided">
+            <li class="p-list__item">Download the ISO image</li>
+            <li class="p-list__item">
+              Create a bootable USB flash drive with <a href="https://etcher.balena.io/">balenaEtcher</a> or similar
+            </li>
+            <li class="p-list__item">Boot from the USB flash drive</li>
+          </ol>
+          <hr class="p-rule" />
+          <p>
+            <a href="/tutorials/install-ubuntu-server">Follow the step-by-step tutorial&nbsp;&rsaquo;</a>
+          </p>
+        </div>
+      </div>
+    </div>
+
+    <div class="row u-sv3">
+      <hr class="p-rule" />
+    </div>
+
+    <div class="row--50-50">
+      <div class="col">
+        <div class="p-section">
+          <h2>Ubuntu {{ releases.latest.full_version }} LTS</h2>
+        </div>
+        <div class="p-image-wrapper u-hide--small u-hide--medium">
+          {{ image(url="https://assets.ubuntu.com/v1/0749ce0e-Orange%20O_no%20background-resized.png",
+                    alt="Oracular Oriole",
+                    width="125",
```

Comment:
The image looks somewhat smaller than the one for the Numbat. Can we make them at least the same width?